### PR TITLE
Post release update to version and npm version dependency updates.

### PR DESCRIPTION
**Pull request by pratiksanglikar**
Updating the vsix version to `0.3.0-alpha` and updating version dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"name": "vscode-tye",
 	"displayName": "Tye",
 	"description": "Makes it easy to debug and interact with applications using Tye.",
-	"version": "0.2.0",
+	"version": "0.2.0-alpha",
```
Comment (philliphoff) [inline package.json:5]: Should be `0.3.0-alpha`?

Comment (pratiksanglikar) [reply package.json]: Ohh right!
Updated now.

File: package.json [hunk 2]
```diff
@@ -369,18 +369,18 @@
...
+		"@types/mocha": "^8.2.3",
+		"@types/node": "^14.17.5",
+		"@types/node-netstat": "^1.8.1",
 		"@types/ps-tree": "^1.1.0",
-		"@types/semver": "^7.3.6",
-		"@types/terser-webpack-plugin": "^5.0.3",
-		"@types/vscode": "^1.57.0",
+		"@types/semver": "^7.3.7",
+		"@types/terser-webpack-plugin": "^5.0.4",
+		"@types/vscode": "^1.58.0",
```
Comment (philliphoff) [inline package.json:383]: If you increment the version of the VS Code type definitions, also increment the VS Code engine version (to ensure we're not trying to use types that exist only on a newer version).

File: package.json [hunk 3]
```diff
@@ -369,18 +369,18 @@
...
+		"@types/glob": "^7.1.4",
+		"@types/gulp": "^4.0.9",
 		"@types/gulp-sourcemaps": "^0.0.34",
-		"@types/js-yaml": "^4.0.1",
-		"@types/mocha": "^8.2.2",
-		"@types/node": "^14.16.0",
-		"@types/node-netstat": "^1.8.0",
+		"@types/js-yaml": "^4.0.2",
+		"@types/mocha": "^8.2.3",
+		"@types/node": "^14.17.5",
```
Comment (philliphoff) [inline package.json:378]: I usually try to match the version of Node used by VS Code (i.e. Electron), to ensure we don't try to use runtime features that don't exist in older versions (and Electron always trails the latest Node.js versions).

File: package.json (hunk 2 above)
Comment (pratiksanglikar) [reply package.json:383]: Thanks for pointing out this. I just used `npm update` and didn't know of these gotchas!

File: package.json (hunk 3 above)
Comment (pratiksanglikar) [reply package.json]: Thanks for pointing out this. I just used `npm update` and didn't know of these gotchas!

**Review by philliphoff — APPROVED** [review]

**Pull request merged**